packages/@glimmer/syntax/lib/generation/printer.ts

```diff
@@ -45,8 +54,8 @@ export interface PrinterOptions {
  * Examples when false:
  *  - Link (component)
  */
-function isVoidTag(tag: string): boolean {
-  return !!(voidMap[tag.toLowerCase()] && tag[0]?.toLowerCase() === tag[0]);
+export function isVoidTag(tag: string): boolean {
+  return !!(voidMap.has(tag.toLowerCase()) && tag[0]?.toLowerCase() === tag[0]);
```

Comment:
`!!` become useless.

Reply:
thanks! I wasn't even looking at that -- more ancient code